src/shared/components/hero-section/index.js

```diff
@@ -99,10 +104,18 @@ class Hero extends Component {
       { count: formattedDownloads }
     )
 
-    const info = {
-      currentVersionStr,
-      latestUpdateDateStr,
-      downloadsStr
+    if (isMobile) {
```

Comment:
If you want to save lines (optional):

```js
const info = isMobile ?
  { `${currentVersionStr} | ${latestUpdateDateStr}`, downloadsStr } :
  { currentVersionStr, latestUpdateDateStr, downloadsStr }
```

Reply:
I think you missed `versionAndDateStr:`
It should be: 
```js
const info = isMobile
      ? { versionAndDateStr: `${currentVersionStr} | ${latestUpdateDateStr}`, downloadsStr }
      : { currentVersionStr, latestUpdateDateStr, downloadsStr }
```
But I got it. Thank you for your suggestion!